src/helper_input.R

```diff
@@ -36,9 +36,21 @@ collapseStory <- function() {
     ), # bsCollapsePanel
     bsCollapsePanel(
       title = "2. Pre-existing Conditions",
-      checkboxInput('is_sick', div("I have ", tags$a("flu-like symptoms", href = urls$cdc_symptoms))),
+      checkboxInput('is_sick', div("I have ", tags$a("symptoms of COVID", href = urls$cdc_symptoms))),
+      
+      conditionalPanel(
+        condition = "input.is_sick == true",
+        checkboxGroupInput("symptoms", "Symptoms",
+                           c("Loss of smell and taste" = "is_loss_smell_taste",
+                             "Severe or significant persistent cough" = "is_cough",
+                             "Severe fatigue" = "is_fatigue",
+                             "Skipped meals" = "is_skip_meal"
```

Comment:
@pendlrobinson "Skipped meals" in itself is a bit hard to understand, can we change this label to "Loss of appetite, skipped meals"?